synapse/models/infotech.py

```diff
@@ -1060,11 +1097,11 @@ def getModelDefs(self):
 
                     ('manu', ('str', {}), {
                         'deprecated': True,
-                        'doc': 'Please use :hardware:make.'}),
+                        'doc': 'Please use :hardware::manufacturer:name.'}),
```

Comment:
Am I missing something? I don't see `:manufacturer:name` on `it:prod:hardware`.

Reply:
![Screenshot from 2024-06-18 09-37-39](https://github.com/vertexproject/synapse/assets/2624245/8fe7d06e-15d3-4060-9076-771dff7d4107)